models/games.models.js

```diff
@@ -1,6 +1,7 @@
 const db = require("../db/connection.js");
 const { sort } = require("../db/data/test-data/categories.js");
 const {checkCategory, checkReviewsById} = require("../db/seeds/utils")
+const jsonObject = require("../endpoints.json");
```

Comment:
Unsure why this is required in here - we shouldn't need the models for this task at all